golang/internal/grpc/grpc.go

```diff
@@ -567,19 +624,19 @@ func executeDeleteContainer(ctx context.Context, req *agent.ContainerDeleteReque
 	}
 }
 
-func executeDeleteMultipleContainers(ctx context.Context, req *common.DeleteContainersRequest, deleteFn DeleteContainersFunc) {
-	if deleteFn == nil {
-		log.Error().Msg("Delete function not implemented")
-		return
-	}
-
-	log.Info().Msg("Deleting multiple containers")
-
-	err := deleteFn(ctx, req)
-	if err != nil {
-		log.Error().Stack().Err(err).Msg("Failed to delete multiple containers")
+func mapDeleteContainersErrorToCommandError(err *agent.AgentError) *agent.AgentCommandError {
```

Comment:
These mappings could be on our "CustomError" type since each struct has its own, that  would make sense. The function then expects a type that implements an interface that has Error() and CommandError() functions. That would be a bit cleaner, what do you think?